packages/radio/src/css/index.js

```diff
@@ -2,29 +2,26 @@ import core from '@pluralsight/ps-design-system-core'
 import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
 
 export default {
-  // group
-
   '.psds-radio-group:focus': {
     outline: 'none'
   },
   '.psds-radio-group--disabled': {
     opacity: '0.5'
   },
 
-  // __button-container
```

Comment:
do we want to drop this convention?

Reply:
yes, I think so.  Specifically, keep the newline, but remove the comment.